itests/org.jboss.tools.hibernate.ui.bot.test/out.txt

```diff
@@ -0,0 +1,801 @@
+Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T17:41:47+01:00)
```

Comment:
Is this file really needed to be committed? I'm not aware of its purpose but looks like it should be added to .gitignore.